src/plugin/reflection_class.cc

```diff
@@ -19,7 +19,11 @@ namespace chapel {
     printer->Print("\n");
     printer->Indent();
 
-    printer->Print("use Encoding;\n");
+    printer->Print(
+      "use Fields;\n"
+      "use RepeatedFields;\n"
+      "use WireEncoding;\n"
+      "use List;\n");
```

Comment:
This is fine for now but I expect we will migrate these to one module (maybe with submodules) and then this would just e.g. `use ProtobufProtocolSupport;`.

Reply:
Yes, that sounds good to me. I will keep that in mind :)